content/micropython/00.first-steps/03.runtime-package/runtime-package.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Arduino MicroPython Runtime"
+description: "Learn how to use the Arduino MicroPython runtime package, which allows you to write MicroPython code in a familiar Arduino style."
+author: "Karl Söderby"
+tags: [MicroPython, Runtime]
+micropython_type: test
+---
+
+The [Arduino Runtime Package](https://github.com/arduino/arduino-runtime-mpy/tree/main) is a MicroPython package that allows you to write and program your board using the classic `setup()` and `loop()` constructs.
```

Comment:
change: "Arduino Runtime for MicroPython is a library that"

suggestion: "[...] using the familiar [...]"

suggestion: "[...] construct. It also adds a few helpers along the way."